modules/caddytls/tls.go

```diff
@@ -255,6 +265,7 @@ func (t *TLS) Provision(ctx caddy.Context) error {
 		OCSP: certmagic.OCSPConfig{
 			DisableStapling: t.DisableOCSPStapling,
 		},
+		DisableStorageCheck: t.DisableStorageChecks,
```

Comment:
Typo, causing test failures

```suggestion
		DisableStorageCheck: t.DisableStorageCheck,
```

Are you not using an IDE with Go linting? It should have warned you about that before you made your commit :thinking: I recommend VSCode with the official Go extension, works very well.

Reply:
I am using vscode ... but, sometimes I think I'm faster in `git commit && git push` than autosaving and linting is. Sorry :/